Automattic/vip-cli
# Add Cache Purge Command

## Description

Customers could use the CLI to purge cache as a similar command used on WP sites.

## Steps to Test

1. Check out PR.
1. Run `npm run build` and `npm link`
1. Run `node ./dist/bin/vip @4716 cache purge-url https://luiztiago-test-node.go-vip.net/`
1. Verify `Purged URLs` success message
